packages/core/src/App/Containers/NotificationsDialog/notifications-dialog.jsx

```diff
@@ -5,147 +5,154 @@ import { CSSTransition } from 'react-transition-group';
 import { Button, DesktopWrapper, Icon, MobileDialog, MobileWrapper, ThemedScrollbars, Text } from '@deriv/components';
 import { BinaryLink } from 'App/Components/Routes';
 import { connect } from 'Stores/connect';
-import { localize } from '@deriv/translations';
+import { localize, Localize } from '@deriv/translations';
 import { toTitleCase, isEmptyObject, isMobile } from '@deriv/shared';
 
 import { EmptyNotification } from 'App/Components/Elements/Notifications/empty-notification.jsx';
 
-class NotificationsDialog extends React.Component {
-    setWrapperRef = node => {
-        this.wrapper_ref = node;
-    };
+const NotificationsListEl = ({ notifications, toggleDialog }) => (
+    <React.Fragment>
+        {notifications && notifications.length ? (
+            notifications.map((item, idx) => (
+                <div className='notifications-item' key={idx}>
```

Comment:
Since notifications have unique keys, it's recommended to use those rather than the index. 

```JS
<div className='notifications-item' key={item.key}>
```

Reply:
Thanks for the comment, 
I was not sure if keys are unique in this list.